packages/victory-core/src/index.d.ts

```diff
@@ -544,7 +551,10 @@ export interface VictoryDatableProps {
   data?: any[];
   dataComponent?: React.ReactElement;
   domain?: DomainPropType;
+  domainPadding?: DomainPaddingPropType;
+  samples?: number;
```

Comment:
unsure if `samples` should be in `VictoryDatableProps` or in the above `VictoryCommonProps`

Reply:
It belongs in `VictoryDatableProps` I think

Reply:
ok great. i will keep this `samples` prop in here and remove the one in `VictoryCommonProps`